content/influxdb/v2.0/process-data/manage-tasks/task-run-history.md

```diff
@@ -44,3 +46,22 @@ influx task run list --task-id=0000000000000000
 {{% note %}}
 Detailed run logs are not currently available in the `influx` CLI.
 {{% /note %}}
+
+## Retry failed task runs
+Use the [`influx task retry-failed` command](/influxdb/v2.0/reference/cli/influx/task/retry-failed/)
+to retry failed task runs.
+
+```sh
+# Retry failed tasks for a specific task
+influx task retry-failed \
+  --id 0000000000000000
+
+# View runs that will retry before actually retrying
```

Comment:
The description from below might be better here: "Print information about runs that would be retried"